Documentation/CrossPlatform/OculusQuestMRTK.md

```diff
@@ -35,6 +35,9 @@ work is 19.1. Older versions can be found from this [archive](https://developer.
 1. Navigate to Mixed Reality Toolkit > Utilities > Oculus > Configure CSC File for Oculus. Updating the csc file filters out the obsolete warnings produced by the Oculus Integration Assets. 
 The MRTK repo contains a csc file that converts warnings to errors, this conversion halts the MRTK-quest configuration process.
 
+1. Navigate to Mixed Reality Toolkit > Utilities > Oculus > Integrate Oculus Integration Unity Modules. Doing this will update the asmdefs with definitions and references needed for the
+relevant Oculus Quest code to function.
+
```

Comment:
nit: It helps to put a screenshot of the menu item to indicate the importance of the new step.  A picture of a step makes it hard to miss.